### ISSUE TITLE
License copyright information

### ISSUE THREAD

**Issue by svengato** [legumeinfo/web-components#1]
When we use this new Apache 2.0 license template, do we fill in the line
`Copyright [yyyy] [name of copyright owner]`
with NCGR-specific information, like
`Copyright 2021 National Center for Genome Resources`
? I am adding the license and code of conduct to Funnotate, but as far as I can tell, this is the only other repository that uses the Apache license and it retains the generic language.

**Comment by alancleary** [comment]
I don't think we ever discussed this. Honestly, I just include a copy of the license with my code so I never bother to fill in our information since I'm not adding the boilerplate that contains it to any of my source files.

I think @adf-ncgr and @cann0010 would be the authorities on this, though.

**Comment by svengato** [comment]
I left it as is, we can always add this information later.

**Comment by adf-ncgr** [comment]
If I understand the question, it is whether the LICENSE file that gets copied from the template repository when a new repo is built from it should have copyright info added. It seems to me that it should, and I think putting NCGR is appropriate since I believe USDA folks can only contribute, but not hold title to repositories that are not in the public domain (@nathanweeks may also be an authority here).

**Comment by alancleary** [comment]
> If I understand the question, it is whether the LICENSE file that gets copied from the template repository when a new repo is built from it should have copyright info added.

I think it was more general, i.e. what should the copyright info in an Apache license be, whether the copy of the license came from the template repo or is being added manually. I think the year the project was created and "National Center for Genome Resources (NCGR)" is reasonable (and should also be added to the template repo's copy of the license).

I want to point out that the "fill in your information here" part of the license we're talking about here is a bit of boilerplate in the appendix that's intended to be copied into source code files as a notification of the licensing, i.e. it doesn't change the meaning or legality of the license if left as `Copyright [yyyy] [name of copyright owner]`. In fact, the appendix it's in could be completely removed from the license.
